src/models/Factor.js

```diff
@@ -207,7 +207,7 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
         }
 
         // MFA_CHALLENGE
-        else if (this.get('canUseResend') && transaction.resend) {
+        else if (this.get('canUseResend') && !this.get('answer') && transaction.resend) {
```

Comment:
What I'm trying to figure out is why you need to check if ```answer``` is null. At the end, if the user types something in the answer textbox or not, it has nothing to do with the user clicking the resend button. So I'm confused on why you need to add ```!this.get('answer') ``` to the conditions.